cirq-core/cirq/experiments/readout_confusion_matrix_test.py

```diff
@@ -17,6 +17,35 @@
 import pytest
 
 from cirq.experiments.single_qubit_readout_calibration_test import NoisySingleQubitReadoutSampler
+from cirq.experiments.readout_confusion_matrix import TensoredConfusionMatrices
+
+
+def add_readout_error(
+    measurements: np.ndarray,
+    zero_errors: np.ndarray,
+    one_errors: np.ndarray,
+    rng: np.random.Generator = np.random.default_rng(),
```

Comment:
prefer to use a `None` default value.  the default value of an argumet is resolved only once ... that means that all places that call the method without specifiying an `rng` will use the same object. with `None` we create a new object for each of these calls which prevents any possible correlation between different calls

```suggestion
    rng: Optional[np.random.Generator] = None,
...
   if rng is None: rng = np.random.default_rng()
```